README.adoc

```diff
@@ -30,28 +30,25 @@ The Planner build can be quickly run using the link:scripts/run-planner.sh[launc
 
 You can use the launcher script to run the Planner either as a Standalone component or as an Integrated component using an external webapp like the fabric8-ui.
 
-- To run a minimal runtime implementation of the Planner using an in-memory mock database use:
+. *Without Docker:* To run a minimal runtime implementation of the Planner using an in-memory mock database use:
 +
+[sh]
 ----
 $ scripts/run-planner.sh --standalone
 ----
 You can access the service on *`localhost:8080`*
++
+NOTE: To run the script on macOS, install `gnu-getopt`, run `brew install gnu-getopt`, and set it in your PATH:
+ `echo 'export PATH="/usr/local/opt/gnu-getopt/bin:$PATH"' >> ~/.zshrc`.
 
-- To run the Planner in an integrated environment using an external webapp, use:
+. *With Docker:* To run the complete Planner stack using *_Docker Compose_* with Core, Auth and DB setups use:
```

Comment:
We have not mentioned Core, Auth earlier in the doc. From a new contributor perspective, I would suggest a line or two explaining how all of them go together in OSIO, in the introduction or here, since you use them in many places in the docs.

Global: Replace DB with database.